content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+To migrate from Sensu Enterprise, see [Sensu Enterprise migration guide][30].
+
+- [Learn about Sensu Go](#learn-about-sensu-go)
+- [Install Sensu Go](#install-sensu-go)
+	- [1. Install the Sensu Go backend](#1-install-the-sensu-go-backend)
+	- [2. Log in to the Sensu web UI](#2-log-in-to-the-sensu-web-ui)
+	- [3. Install sensuctl on your workstation](#3-install-sensuctl-on-your-workstation)
+	- [4. Set up Sensu users](#4-set-up-sensu-users)
+	- [5. Install agents](#5-install-agents)
+- [Translate your configuration](#translate-your-configuration)
+	- [1. Run the translator](#1-run-the-translator)
+	- [2. Translate checks](#2-translate-checks)
+	- [3. Translate filters](#3-translate-filters)
+	- [4. Translate handlers](#4-translate-handlers)
+	- [5. Upload your config to your Sensu Go instance](#5-upload-your-config-to-your-sensu-go-instance)
+- [Activate assets](#activate-assets)
+- [Sunset your Sensu 1.x instance](#sunset-your-sensu-1x-instance)
+- [Resources](#resources)
+
+## Learn about Sensu Go
+
+Sensu Go includes powerful features to make monitoring easier to build, scale, and offer as a self-service tool to your internal customers. 
+Read the [blog post][31] for more information about the features of Sensu Go.
+
+To set up a local Sensu Go playground, download and start the [Sensu sandbox][32] using Vagrant and Virtual box.
+Get started with Sensu by running through the [sandbox tutorial][32].
+
+## Install Sensu Go
+
+Sensu is now provided as three packages: sensu-go-backend, sensu-go-agent, and sensu-go-cli (sensuctl).
+This results in a fundamental change in Sensu terminology from Sensu Core 1.x: the server is now the backend; the client is now the agent.
+
+To install Sensu Go alongside your current Sensu instance, first [upgrade][33] to at least [Sensu Core 1.7.0][34].
+
+#### Sensu Go architecture
+
+<img src="/images/install-sensu.svg" alt="Sensu architecture diagram">
+
+Powered by an an embedded transport and [etcd][16] datastore, the **Sensu backend** gives you flexible, automated workflows to route metrics and alerts.
+Sensu backends require persistent storage for their embedded database, disk space for local asset caching, and three exposed ports:
+
+- `3000` - Sensu [web UI][13]
+- `8080` - Sensu [API][41] used by sensuctl, some plugins, and any of your custom tooling
+- `8081` - WebSocket API used by Sensu agents
+
+Sensu backends running in a [clustered configuration][14] require additional ports.
+See the [deployment guide][deploy] and [hardware requirements][hardware] guide for deployment recommendations.
+
+**Sensu agents** are lightweight clients that run on the infrastructure components you want to monitor.
+Agents register automatically with Sensu as entities and are responsible for creating check and metric events to send to the backend event pipeline.
+Optionally, agents can expose ports `3031` for the [agent API][50] and `8125` for the [StatsD listener][51].

```

Comment:
You may want to mention to client compatibility port at 3030.  You reference this a little later as something to disable in a side-by-side migration.

Reply:
Fixed